src/app/results/results.component.html

```diff
@@ -75,7 +75,7 @@
                 <ul class="pagination" id="pag-bar">
                     <li class="page-item"><span class="page-link" href="#" (click)="decPresentPage()">Previous</span></li>
                     <li class="page-item" *ngFor="let num of getNumber(maxPage)"><span class="page-link" *ngIf="presentPage>=4 && num<=noOfPages" [class.active_page]="getStyle(presentPage-3+num)"
-                            (click)="getPresentPage(presentPage-3+num)" href="#">{{presentPage-3+num}}</span>
+                            (click)="getPresentPage(presentPage-4+num)" href="#">{{presentPage-3+num}}</span>
```

Comment:
subtract -1 here https://github.com/fossasia/susper.com/blob/master/src/app/results/results.component.ts#L52 from `this.presentPage` because this would be meaning full and doesn't remove the significant meaning of return values of the function.